examples/data_services/geocoding_dataframe.ipynb

```diff
@@ -194,9 +177,9 @@
        "  <link href=&quot;https://fonts.googleapis.com/css?family=Roboto&quot; rel=&quot;stylesheet&quot; type=&quot;text/css&quot;>\n",
        "\n",
        "  <!-- External libraries -->\n",
-       "  \n",
-       "  <!-- base64-js -->\n",
-       "  <script src='https://libs.cartocdn.com/cartoframes/dependencies/base64.js'></script>\n",
+       "\n",
+       "  <!-- pako -->\n",
```

Comment:
https://www.youtube.com/watch?v=QZaqZIbJiKo

Reply:
TBH I don't know where it comes from 😂